### ISSUE TITLE
Unable to create Database in AWS

### ISSUE THREAD

**Issue by charulaks** [orientechnologies/orientdb-studio#238]
I have installed OrientDB in my instance of AWS and I am not able to create a new database from the web console. My root access is not getting authenticated at all. I used the password specified in the server config xml. However, using the same root password, I am able to view the server management page though. Please help me!

**Comment by lvca** [comment]
Can you copy & paste your console session?

**Comment by charulaks** [comment]
First of all, thanks a million for such a quick reply! Here is what I did for the issues:

Issue #1: could not create database from the web console
    Ans: Used putty to log on to the server and created the DB manually

Issue #2: when I did "./console.sh", I got a never ending repetitive errors which stop with ctrl+C.. For this, the solution was to upgrade to Orientdb 2.x which is not available in AWS
    Ans: I ran the console.sh with sudo.. "sudo ./console.sh" and then logged in with root credentials. It would be great to know how to get the latest OrientDB in AWS

Somehow I am able to create a database now. Could you please tell me the best way to 
1. how to change server config xml in aws?
2. how to update OrientDB in AWS? 

could not find any docs or online help pointing to my questions. Please help me find the answers.

**Comment by StrongAmerica** [comment]
Hi Luca,
I have OrientDB version 2.0.1 and have instance on AWS where they have OrientDB 1.7.9
For most of the issues I am getting error saying version mismatch. Could please let me know how can I update orientDB version in AWS ? I am also raising new request with specific errors. Thank you

**Comment by Archana14** [comment]
Hi Luca,
I have OrientDB version 2.0.1 and have instance on AWS where they have OrientDB 1.7.9
For most of the issues I am getting error saying version mismatch. Could please let me know how can I update orientDB version in AWS ? I am also raising new request with specific errors. Thank you

**Comment by lvca** [comment]
Did you resolve?

**Comment by charulaks** [comment]
Yes. For the benefit of other googlers, here is what I did:

1. I removed the instance of Orientdb that was at the AWS market place
2. Set up an EC2 instance and copied the latest production worthy .tar.gz of OrientDB available. Since there is no access to /opt at EC2 instance, I had to copy it to HOME using winSCP
3. extract and run the server.sh

The first time around, running the server or the console, or creating a new databse throws exceptions. But logout and login again and try again and it will work.
